### DISCUSSION TITLE
How to delete custom printers

### DISCUSSION THREAD

**Discussion by olaab** [OrcaSlicer/OrcaSlicer#5124] [Q&A]
Yes I can't find how to remove, delete my custom printers I made.
OrcaSlicer V2.0.0, Win 10.

**Comment by LJKraus** [comment]
Select the preset, click the edit icon, click the "X" next to the save preset icon.
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/38ed406d-2f49-4f62-9c5f-e5a8a1d66798)

**Reply by olaab** [reply]
Thank you :-)

**Reply by ghost** [reply]
I don't have the x (or in some videos, it's a -) button. It's a magnifying glass in my version. It allows for searching within settings.

For others that don't see it, you need to enable advanced switch, close the window, and click the edit button again for the - or x button to appear.
